keras_cv/models/object_detection/faster_rcnn.py

```diff
@@ -487,9 +488,8 @@ def compute_loss(self, images, boxes, classes, training):
         )
 
     def train_step(self, data):
-        images, y, sample_weight = tf.keras.utils.unpack_x_y_sample_weight(data)
-        if sample_weight is not None:
-            raise ValueError("`sample_weight` is currently not supported.")
+        images, y = unpack_input(data)
```

Comment:
One issue here: sample weight.  I guess we just don’t support it with the dict input?

Reply:
Correct. (Although this is not a regression -- we already don't support sample weight in the OD models)